tests/test_cvp.py

```diff
@@ -26,3 +27,16 @@ def test_cvp():
         v1 = tuple((v1*A)[0])
 
         assert v0 == v1
+
+
+def test_cvp_too_large():
+    from fpylll.config import max_enum_dim
+    m = max_enum_dim + 1
+    n = max_enum_dim + 1
+    A = make_integer_matrix(m, n)
+    A = LLL.reduction(A)
+    M = GSO.Mat(A)
+    M.update_gso()
+    t = list(make_integer_matrix(n, n)[0])
+    with pytest.raises(NotImplementedError):
+        CVP.closest_vector(A, t)
```

Comment:
Do we want to sanity check the output somehow ? Like; it is in the lattice and it is somewhat short indeed (say 2*GH) ?

Reply:
That particular test just raises a `NotImplementedError` since the dimension is beyond the maximum enumeration dimension.